src/main/java/com/google/api/codegen/py/PythonImportHandler.java

```diff
@@ -59,14 +61,15 @@ public PythonImportHandler(Interface service) {
     addImportExternal("google.gax", "path_template");
 
     // Add method request-type imports.
-    for (Method method : service.getMethods()) {
+    for (MethodConfig methodConfig : apiConfig.getInterfaceConfig(service).getMethodConfigs()) {
+      Method method = methodConfig.getMethod();
       addImport(
           method.getFile(),
           PythonImport.create(
               ImportType.APP,
               method.getFile().getProto().getPackage(),
-              PythonProtoElements.getPbFileName(method.getInputMessage())));
-      for (Field field : method.getInputType().getMessageType().getMessageFields()) {
+              PythonProtoElements.getPbFileName(method)));
```

Comment:
Ah I see. This makes sense. I'll change this to create the input from the inputMessages protofile.